### PR TITLE
Change regex not letting utf-8 garden names save

### PR DESCRIPTION
**Pull request by jenkr55**
Closes https://github.com/Growstuff/growstuff/issues/1634

I removed the regex that wasn't letting UTF-8 chars save.

### CODE REVIEW

**Comment by Br3nda** [comment]
i like this. Anyone know why we stopped it? Was it just because it goes in the url?

**Comment by pozorvlak** [comment]
What happens if you create a garden containing a character which is [not allowed in URLs](https://stackoverflow.com/questions/1856785/characters-allowed-in-a-url)? Is Rails' slug-making algorithm smart enough to escape them?

**Comment by Br3nda** [comment]
If we allow new lines in garden names, what is the consequences? I think just the csv render types -- would we render the new lines on pages? they don't really make sense to me to have a new line in the garden name.

**Comment by jenkr55** [comment]
Looks like it's smart enough to not enter newlines and other breaking things in CSV format (just the actual characters for "\n" etc) and URLs seem fine, but to be safe I added a regex (that also allows utf-8)

**Comment by Br3nda** [comment]
merge when ready

File: spec/models/garden_spec.rb [hunk 1]
```diff
@@ -34,19 +34,24 @@
...
   end
 
-  it "doesn't allow a new lines in garden names" do
+  it "doesn't allow a new line chars in garden names" do
     garden = FactoryBot.build(:garden, name: "My garden\nI am a 1337 hacker")
     garden.should_not be_valid
   end
 
+  it "doesn't allow a new lines in garden names" do
+    garden = FactoryBot.build(:garden, name: 'My garden\nI am a 1337 hacker')
```
[the remark this answers is not in the document]
Comment (Br3nda) [reply spec/models/garden_spec.rb]: Good catch!

**Pull request merged**